examples/rosetta/LeastSquares/main.roc

```diff
@@ -0,0 +1,29 @@
+app "rosetta-example"
+    packages { pf: "https://github.com/roc-lang/basic-cli/releases/download/0.3.1/97mY3sUwo433-pcnEQUlMhn-sWiIf_J9bPhcAFZoqY4.tar.br" }
+    imports [
+        pf.Stdout,
+    ]
+    provides [main] to pf
+
+main =
+    nStr = findNumber 1 |> Num.toStr
+
+    Stdout.line "The least positive integer n, where the difference of n*n and (n-1)*(n-1) is greater than 1000, is \(nStr)"
+
+## A recursive function that takes an `U32` as its input and returns the least
+## positive integer number `n`, where the difference of `n*n` and `(n-1)*(n-1)`
+## is greater than 1000.
+##
+## The input `n` should be a positive integer, and the function will return an
+## `U32`representing the least positive integer that satisfies the condition.
+##
+findNumber : U32 -> U32
```

Comment:
I think this function should should be `{} -> U32` the parameter has no meaning to the caller. Instead, this should have an inner helper function that is what this function is currently.